mibidata/constants.py

```diff
@@ -13,3 +13,91 @@
     'Blue': (0.0, 0.0, 1.0),
     'Gray': (1.0, 1.0, 1.0),
 }
+
+OVERLAY_MIN_BRIGHTNESS = -0.9
+
+# The kernels below are those used by WegGL convolution in MIBItracker.
+# pylint: disable=line-too-long
+OVERLAY_SMOOTHING_KERNELS = [
+    [
```

Comment:
the first array here (a blur value of 0, seeming no blur) appears to be out of order. If this is the desired kernel, it would be good to add some explanation as to why.